README.md

```diff
@@ -25,9 +26,22 @@ before publishing to operatorhub.io.
 
 Make the unreleased bundles available in Marketplace by adding the app registry:
 ```bash
+# Remove the hco-bundle from the community-operators sources
+$ kubectl get operatorsource -n openshift-marketplace community-operators -o yaml | sed "s/hco-operatorhub,//" | kubectl apply -f -
+$ kubectl get catalogsourceconfig -n openshift-marketplace community-operators  -o yaml | sed "s/hco-operatorhub,//" | sed "s/hco-operatorhub\:*,//" | kubectl apply -f -
+
+# Add the unreleases bundle source
 $ curl https://raw.githubusercontent.com/kubevirt/hyperconverged-cluster-operator/master/tools/quay-registry.sh | bash -s $QUAY_USERNAME $QUAY_PASSWORD
```

Comment:
add `jq` as a requirement

```
bash: line 38: jq: command not found
(23) Failed writing body
```

Reply:
I want to limit the steps in the README.  I'll leave it to the user to `yum install jq`